src/components/Proposal/ActionButton.tsx

```diff
@@ -90,12 +91,12 @@ class ActionButton extends React.Component<IProps, IState> {
       daoEthBalance,
       detailView,
       proposalState,
-      redeemProposal,
       rewardsForCurrentUser,
     } = this.props;
 
     const executable = proposalEnded(proposalState) && !proposalState.executedAt;
     const expired = closingTime(proposalState).isSameOrBefore(moment());
+    const _handleRedeemProposal = this.handleRedeemProposal.bind(this);
```

Comment:
The pattern we use in other places (and I think is slightly better) is to do the binding in the consturctor. I.e. the constructor would contain the line:

`this.handleRedeemProposal = this.handleRedeemProposal.bind(this)`

And then you can use the bound `this.handleRedeemProposal` anywhere (not just in render)

Reply:
I'm reflexively suspicious of that pattern, but anyway went ahead and made it so.